src/Registry/registry_instance.jl

```diff
@@ -53,6 +54,12 @@ struct PkgInfo
 
     # Deps.toml
     deps::Dict{VersionRange, Dict{String, UUID}}
+
+    # WeakCompat.toml
+    glue_compat::Dict{VersionRange, Dict{String, VersionSpec}}
```

Comment:
So there will be new registry files written for these then? i.e. we'll need to update RegistryTools.jl to support these as well?